Meadowlark-js/backends/meadowlark-postgresql-backend/src/repository/Update.ts

```diff
@@ -34,6 +35,13 @@ export async function updateDocumentById(
   try {
     await client.query('BEGIN');
 
+    const recordExistsResult = await client.query(findAliasIdsForDocumentSql(id));
+
+    if (recordExistsResult.rowCount === 0) {
```

Comment:
Bug that Stephen found, without checking for the existence of a record on an update, reference and alias table insertions but no document table insert

Reply:
Give yourself credit here! I saw it, but didn't know what was going on and moved past. You saw it and dug into the details.